testfiles/10xSuite/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#
+
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgFullServiceRestaurantNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgHospitalNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgLargeHotelNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgLargeOfficeNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgMediumOfficeNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgMidriseApartmentNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgOutPatientNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgPrimarySchoolNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgQuickServiceRestaurantNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgSecondarySchoolNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgSmallHotelNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgSmallOfficeNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgStand-aloneRetailNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgStripMallNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgSuperMarketNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgWarehouseNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_VetBuilding.idf VetBuilding.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_100_zones_with_HVAC_London_Db.idf 100_zones_London.Gatwick.037760_IWEC.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_Bay_Gerrard_WWR_82_Toronto.idf Bay_CAN_ON_Toronto.716240_CWEC.epw)
+#ADD_SIMULATION_TEST(IDF_FILE Bay_Gerrard_WWR_82_Toronto.idf.idf VetBuilding.epw)
```

Comment:
@XuanLuoLBL It kinda looked like this line was an unintentional addition.  The file is already added as a test in the line above, and the epw sorta doesn't match.  Can you verify, and if so go ahead and remove this line?  Thanks!

Reply:
this was a typo. fixed.